boot/seal.go

```diff
@@ -74,12 +74,17 @@ type FDESetupHookParams struct {
 	Key     secboot.EncryptionKey
 	KeyName string
 
-	Models []*asserts.Model
-
 	//TODO:UC20: provide bootchains and a way to track measured
 	//boot-assets
 }
 
+// FdeSetupHookResult contains the outputs of the fde-setup hook
+type FDESetupHookResult struct {
+	// result when called with "initial-setup"
+	EncryptedKey []byte           `json:"encrypted-key"`
```

Comment:
@pedronis I just talked to Ondra and it's still possible to update their v2 hook - happy to change this to a better name.

Reply:
I think this needs to be sealed-key (and match Ondra's code), until we know what we can/can't do about the reveal side